src/afl-fuzz-run.c

```diff
@@ -402,6 +404,8 @@ u8 calibrate_case(afl_state_t *afl, struct queue_entry *q, u8 *use_mem,
 
         }
 
+        if (!var_detected) WARNF("instability detected during calibration\n");
```

Comment:
This is particularly useful if you expect a testcase to be 100% deterministic and it isn't.

Reply:
you need to check too if he user us NOT using the UI. because otherwise this will f*ck up the UI view.

Reply:
It appears that `from_queue` is only true when not in the UI. Is there a different way to detect that?

Reply:
`if (afl->afl_env.afl_no_ui && !var_detected)  { WARNF() ; }`